lib/client.js

```diff
@@ -379,6 +377,8 @@ Client.prototype.query = function (config, values, callback) {
         rejectOut = reject
       })
       query.callback = (err, res) => err ? rejectOut(err) : resolveOut(res)
+    } else {
+      result = query
```

Comment:
I need a reference to the query to be able to cancel it later. Is this breaking something?